src/annotator/guest.coffee

```diff
@@ -140,7 +129,7 @@ module.exports = class Guest extends Delegator
 
     return Promise.all([metadataPromise, uriPromise]).then ([metadata, href]) =>
       return {
-        uri: normalizeURI(href, baseURI),
+        uri: normalizeURI(href),
```

Comment:
The `baseURI` argument is no longer needed here because `normalizeURI` uses the current document's base URI as its default value for the second argument.